cmd/rancher-desktop-guestagent/main.go

```diff
@@ -52,6 +52,7 @@ var (
 		"file path for Containerd socket address")
 	vtunnelAddr             = flag.String("vtunnelAddr", vtunnelPeerAddr, "Peer address for Vtunnel in IP:PORT format")
 	enablePrivilegedService = flag.Bool("privilegedService", false, "enable Privileged Service mode")
+	listenerAddr            = flag.String("listenerAddr", net.IPv4zero.String(), "default service listener address")
```

Comment:
Can we make the variable name a bit more descriptive? `listenerAddr` made sense in the context of `WatchForServices`, however, as a config argument it might be a little too vague. 
e.g `k8sServiceListenerAddr`, `kubeServiceListenerAddr` or `serviceListenerAddr`.

Reply:
Yes, I agree :+1: